tests/test_commands.py

```diff
@@ -686,6 +686,16 @@ def test_config_set(self, r):
         assert r.config_set("timeout", 0)
         assert r.config_get()["timeout"] == "0"
 
+    @skip_if_server_version_lt("7.0.0")
+    @skip_if_redis_enterprise()
+    def test_config_set_multi_params(self, r: redis.Redis):
```

Comment:
Here's a lazy, future option for writing tests like this (psuedocode). It comes from a pattern used in go:
```
x = [('timeout', 70), ('maxmemory', 100)]
r.config_set(*x)
for y in x:
    assert r.config_get(x[0]) == x[1]
```